src/Controls/tests/DeviceTests/Elements/View/ViewTests.cs

```diff
@@ -65,6 +65,17 @@ await AttachAndRun<LayoutHandler>(layout, async (_) =>
 				Assert.Equal(55, frame.Bottom, 0.5d);
 				Assert.Equal(60, desiredSize.Width, 0.5d);
 				Assert.Equal(60, desiredSize.Height, 0.5d);
+
+				control.WidthRequest = 60;
+				
+				await Task.Delay(100);
+				
+				frame = control.Frame;
+				desiredSize = control.DesiredSize;
+				Assert.Equal(5, frame.X, 0.5d);
+				Assert.Equal(60, frame.Width, 0.5d);
```

Comment:
This was failing on `ListView` and `TableView` due to the fact they were not propagating `SetNeedsLayout` to the parent.